### PR TITLE
Add edly_credentials_app with middleware & context processor

### PR DESCRIPTION
**Pull request by iam-mhaseeb**
**Description:** Added edly_credentials_app with middleware & context processor. Currently, we don't need a context processor but it was added keeping the future case in mind we would need it when we will add theming in credentials service. It might also get some tweaks at that time.

**Jira:** https://edlyio.atlassian.net/browse/EDLY-1902
Screenshot 2020-09-22 at 12 50 49 PM

### CODE REVIEW

File: credentials/apps/edx_credentials_extensions/edly_credentials_app/admin.py [hunk 1]
```diff
@@ -0,0 +1 @@
+from django.contrib import admin
```
Comment (rehan99000) [inline credentials/apps/edx_credentials_extensions/edly_credentials_app/admin.py:1]: we don't necessarily need to create an admin file if we are not going to register a model

File: credentials/apps/edx_credentials_extensions/edly_credentials_app/urls.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+from django.conf.urls import url
```
Comment (rehan99000) [inline credentials/apps/edx_credentials_extensions/edly_credentials_app/urls.py:1]: why this file if there are no views?

File: credentials/apps/edx_credentials_extensions/edly_credentials_app/views.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
+"""
```
Comment (rehan99000) [inline credentials/apps/edx_credentials_extensions/edly_credentials_app/views.py:1]: similarly we don't need this file unless there actually is a view

**Review by rehan99000 — CHANGES_REQUESTED** [review]
minor changes otherwise fine.

File: credentials/apps/edx_credentials_extensions/edly_credentials_app/views.py (hunk 3 above)
Comment (iam-mhaseeb) [reply credentials/apps/edx_credentials_extensions/edly_credentials_app/views.py:1]: similarly, here kept it there if in future we need to use this.

File: credentials/apps/edx_credentials_extensions/edly_credentials_app/urls.py (hunk 2 above)
Comment (iam-mhaseeb) [reply credentials/apps/edx_credentials_extensions/edly_credentials_app/urls.py:1]: similarly, here kept it there if in future we need to use this.

File: credentials/apps/edx_credentials_extensions/edly_credentials_app/admin.py (hunk 1 above)
Comment (iam-mhaseeb) [reply credentials/apps/edx_credentials_extensions/edly_credentials_app/admin.py:1]: Kept it there if in future we need to use this.

**Review by rehan99000 — APPROVED** [review]

**Review by zubair-arbi — APPROVED** [review]
LGTM 👍

**Review by burhandodhy — APPROVED** [review]

**Review by etechdevai — APPROVED** [review]

**Pull request merged**